src/main/resources/templates/sites/ranking.html

```diff
@@ -76,8 +76,8 @@ <h2 th:text="#{sites.ranking}"></h2>
 </div>
 <!-- User List -->
 <div class="container-fluid">
-  <div class="shadow-sm" id="table-container" th:if="${allUsers.size()>0}">
-    <table class="user-table table">
+  <div th:if="${allUsers.size()>0}" style="overflow-x:auto;">
+    <table style="width: 80%; margin-left: 10%; border-color: #332f2d" class="user-table table shadow-sm">
```

Comment:
Hey, don't use inline styles, we have stylesheets for that ;)
The HTML get's really messy and CSS debugging is harder if the style is'nt applied at the same file

Reply:
Also save the every color in '_colorscheme.less'